tools/test-setup.sh

```diff
@@ -442,9 +443,12 @@ if [[ "${PODMAN_VERSION}" != 'null' ]] && [[ "${SKIP_PODMAN:-}" != '1' ]]; then
     log notice "Retrieving ansible-lint version from ee"
     EE_ANSIBLE_LINT_VERSION=$(get_version \
         podman run --rm "${IMAGE}" ansible-lint --nocolor --version)
+    log notice "ansible: ${EE_ANSIBLE_VERSION}, ansible-lint: ${EE_ANSIBLE_LINT_VERSION}"
     log notice "Test podman ability to mount current folder with write access, default mount options"
-    podman run --rm -v "$PWD:$PWD" ghcr.io/ansible/creator-ee:latest \
-        bash -c "[ -w $PWD ] && echo 'Mounts working' || { echo 'Mounts not working. You might need to either disable or make selinux permissive.'; exit 1; }"
+    podman run -v "$PWD:$PWD" \
```

Comment:
Try setting user as root here, that might get past the filesystem issue

Reply:
keeping this as is per disscussion